### PR TITLE
ci: build and release for aarch64

### LINKED ISSUE #400: Create tt packages for 64bit ARM (aarch64) Linux

### PR DESCRIPTION
**Pull request by 0x501D**
Closes #400

### CODE REVIEW

File: test/integration/install/test_install.py [hunk 1]
```diff
@@ -300,6 +300,10 @@ def test_install_tarantool(tt_cmd, tmpdir):
     assert os.path.exists(os.path.join(tmpdir, "bin", "tarantool_2.10.7"))
 
 
+@pytest.mark.skipif(
+        os.getenv("TT_SKIP_DOCKER_TESTS") is not None,
```
Comment (psergee) [inline test/integration/install/test_install.py:304]: Use marks for this purpose.

**Review by psergee — COMMENTED** [review]
Thank you for the patch!

File: cli/docker/docker_integration_test.go [hunk 2]
```diff
@@ -18,6 +18,9 @@ import (
 )
 
 func findAndRemoveBuiltImage(t *testing.T, dockerClient *client.Client, expectedTag string) {
+	if os.Getenv("TT_SKIP_DOCKER_TESTS") != "" {
```
[the remark this answers is not in the document]
Comment (0x501D) [reply cli/docker/docker_integration_test.go]: Fixed

File: test/integration/install/test_install.py (hunk 1 above)
Comment (0x501D) [reply test/integration/install/test_install.py]: Fixed

File: .github/workflows/full-ci.yml [hunk 3]
```diff
@@ -52,6 +52,45 @@ jobs:
...
+        shell: bash
+
+      - name: Install etcd
+        uses: ./.github/actions/setup-etcd
+
+      - name: Build test
+        run: mage build
+
+      - name: Unit tests
+        run: mage unitfullskipdocker
```
Comment (psergee) [inline .github/workflows/full-ci.yml:89]: Please, make it sub-command like for `coverage`, `default` and `full`. Not done for integration yet, but can be done for unit.

**Review by psergee — APPROVED** [review]
Let's remove the alias for unit test without docker. Aliases are only for backward compatibility now (with 'unit' and 'unitfull' commands)
lgtm.

**Review by ylobankov — APPROVED** [review]

**Pull request merged**